animations/index.md

```diff
@@ -0,0 +1,158 @@
+makes it easy to implement a variety of animation types.
+Many widgets, especially
+[Material Design widgets](https://flutter.io/widgets/material/),
+come with the standard motion effects defined in their design spec,
+but it's also possible to customize these effects.
+
+The following resources are a good place to start learning the Flutter
+animation framework. Each of these docs shows, step by step, how
+to write animation code.
+Note that more docs are in the works on how to implement common design patterns,
+such as shared element transitions, and physics-based animations.
+If you have a specific request, please
+[file an issue](https://github.com/flutter/flutter/issues).
+
+* [Tutorial: Animations in Flutter](/tutorials/animation/) <img src="/images/ic_new_releases_black_24px.svg" alt="this doc is new!"> NEW<br>
+Explains the fundamental classes in the Flutter animation package
+(controllers, Animatable, curves, listeners, builders),
+as it guides you through a progression of tween animations using
+different aspects of the animation APIs.
+
+* [Zero to One with Flutter, part
+1](https://medium.com/dartlang/zero-to-one-with-flutter-43b13fd7b354) and [part
+2](https://medium.com/dartlang/zero-to-one-with-flutter-part-two-5aa2f06655cb)<br>
+Medium articles showing how to create an animated chart using tweening.
+
+* [Building Beautiful UIs with
+Flutter](https://codelabs.developers.google.com/codelabs/flutter/index.html#0)<br>
+Codelab demonstrating how to build a simple chat app. [Step 7 (Animate
+your app)](https://codelabs.developers.google.com/codelabs/flutter/index.html#6)
+shows how to animate the new message&mdash;sliding it from the input area up
+to the message list.
+
+## Animation types
+
+Animations fall into one of two categories: tween- or physics-based.
+The following sections explain what these terms mean, and points you to
+resources where you can learn more. Note that, in some cases,
+the best documentation we currently have is example code in the
+Flutter gallery. More docs and examples will be forthcoming.
+
+### Tween animation
+
+Short for _in-betweening_. In a tween animation, the beginning
+and ending points are defined, as well as a timeline, and a curve
+that defines the timing and speed of the transition.
+The framework calculates how to transition from the beginning point
+to the end point.
+
+The docs listed above, such as the [animations
+tutorial](/tutorials/animation/) aren't about tweening,
+specifically, but they use tweens in their examples.
+
+### Physics-based animation
+
+In physics-based animation, motion is modeled to resemble real-world
+behavior. When you toss a ball, for example, where and when
+it lands depends on how fast it was tossed, how heavy it is, and how
+far off the ground. Similarly, dropping a ball attached to a spring falls
+(and bounces) differently than dropping a ball attached to a string.
+
+Note that more docs and examples on physics-based animations will be
```

Comment:
nit: i would remove "note that".

We may want to remove the promises for more documentation in general, actually. If we add more docs, great. If we never do, then these promises aren't helpful.